Controllers/categories.controller.js

```diff
@@ -0,0 +1,11 @@
+const { selectCategories } = require("../Model/categories.model");
+
+exports.getCategories = (req, res, next) => {
+  selectCategories("this")
```

Comment:
What is "this"?  selectCategories doesn't take an argument according to how you've defined it.

Reply:
Was a placholder for temp purposes for when I came back out of the model
has been Adjusted since